rpm/maliit-framework-wayland.spec

```diff
@@ -78,7 +78,7 @@ mkdir -p build
 cd build
 cmake -DCMAKE_INSTALL_PREFIX=/usr \
       -Denable-docs=OFF \
-      -Denable-glib=OFF \
+      -Denable-glib=ON \
```

Comment:
If glib is enabled, in theory it should also do a BuildRequires for it.

Reply:
Makes sense - added
```
BuildRequires:  pkgconfig(glib-2.0)
BuildRequires:  pkgconfig(gio-2.0)
``
to match upstream/CMakeLists.txt

```
    find_package(GLib2)
    find_package(GIO)
```